### ISSUE TITLE
Can you please give a demo to show how to use your code in training pipeline?

### ISSUE THREAD

**Issue by jhgfkdj** [zjysnow/CircleLoss#2]
I try to use your formula in code, but there is a strange problem, when i start the training the loss is not descending. I'm not quite sure where i made some mistakes. So if you kindly show a demo how to use your code in a whole training pipeline？By the way, i am not using gluon but symbol, and i use MakeLoss() and SoftmaxOutput() as my loss construction.

**Comment by zjysnow** [comment]
sorry for late. I have updated a demo for training. If you use Similarity in my code, you should set 'use_bias=False'. I don't know how to use symbol here. sorry!
CircleLoss here you must give the cos similarity to pred!

**Comment by zjysnow** [comment]
you can see this [paper](https://arxiv.org/pdf/2002.10857v2.pdf)

**Comment by jhgfkdj** [comment]
> sorry for late. I have updated a demo for training. If you use Similarity in my code, you should set 'use_bias=False'. I don't know how to use symbol here. sorry!
> CircleLoss here you must give the cos similarity to pred!

Oh thank you for your train_neth. I have implemented my symbol circleloss severdays ago, and of course as you said bias was False and pred was cos similarity. Now i am training it. Thanks for referring to your code, i apprecirate it!
